PontoFacil/PontoFacil/Repositories/Repository.cs

```diff
@@ -1,12 +1,12 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using PontoFacil.Models;
 
 namespace PontoFacil.Repositories
 {
-    class Repository
+    class Repository : IRepository<ClockIn>
     {
+        public void SaveClockIn(ClockIn clockIn)
```

Comment:
Please, check with @gmojunior if this is the right approach when using the repository